### PR TITLE
Nice timeout.

### PR DESCRIPTION
**Pull request by VortexHero**
Elon approved.

### CODE REVIEW

**Comment by HeadCookie** [comment]
Nice

**Comment by VitalWinter** [comment]
Nice

**Comment by Zelakolase** [comment]
LGTM

**Review by sentouki — COMMENTED** [review]
lgtm

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/module/ManhattanClientsModule.scala [hunk 1]
```diff
@@ -50,7 +50,7 @@ object ManhattanClientsModule extends TwitterModule with ConfigUtils {
 
     ManhattanKVEndpointBuilder(client)
       .maxRetryCount(1)
-      .defaultMaxTimeout(Duration.fromMilliseconds(70))
+      .defaultMaxTimeout(Duration.fromMilliseconds(69))
```
Comment (dylan-green) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/module/ManhattanClientsModule.scala:53]: Nice. LGTM!

Comment (s-ven) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/module/ManhattanClientsModule.scala:53]: noice

Comment (BrodaNoel) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/module/ManhattanClientsModule.scala:53]: Oh man... I love internet.
It's going to be soooo boring to be death.

**Review by s-ven — COMMENTED** [review]
&nbsp;

**Pull request closed without merging**